workloads/network-perf/run_network_compare.sh

```diff
@@ -31,12 +31,8 @@ else
   compare_uuid=$2
 fi
 
-git clone https://github.com/cloud-bulldozer/touchstone
-cd touchstone
-python3 -m venv ./compare
-source ./compare/bin/activate
-pip3 install -r requirements.txt
-python3 setup.py develop
+pip3 uninstall -y touchstone
+pip3 install git+https://github.com/cloud-bulldozer/touchstone
```

Comment:
do we not want to use a venv for this install?

Reply:
done